docs/legal/COPYRIGHT

```diff
@@ -0,0 +1,69 @@
+Copyright � 1993, 2018, Oracle and/or its affiliates.
```

Comment:
What's this?

Reply:
Javadoc's autogenerated files.
I've restored the original `docs/legal` files. Let me know if you want to remove them or something else.